dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       {
         "value": "605BZ",
-        "label": "LOMA LINDA VA CLINIC"
+        "label": "Loma Linda VA Clinic"
       },
       {
         "value": "612QD",
-        "label": "HOWE ROAD VA CLINIC"
+        "label": "Howe Road VA Clinic"
       },
       {
         "value": "600",
-        "label": "VA LONG BEACH HEALTHCARE SYSTEM"
+        "label": "Tibor Rubin VA Medical Center"
       },
       {
         "value": "605",
-        "label": "LOMA LINDA HCS"
+        "label": "Jerry L. Pettis Memorial Veterans' Hospital"
       },
       {
         "value": "612",
         "label": "NORTHERN CALIFORNIA HCS"
       },
       {
         "value": "640",
-        "label": "PALO ALTO VA MEDICAL CENTER"
+        "label": "Palo Alto VA Medical Center"
       },
       {
         "value": "662",
-        "label": "SAN FRANCISCO VAMC"
+        "label": "San Francisco VA Medical Center"
       },
       {
         "value": "664",
-        "label": "VA SAN DIEGO HEALTHCARE SYSTEM (664)"
+        "label": "San Diego VA Medical Center"
       },
       {
         "value": "691",
-        "label": "VA GREATER LOS ANGELES HEALTHCARE SYSTEM - WEST LOS ANGELES DIVISION"
+        "label": "West Los Angeles VA Medical Center"
       },
       {
         "value": "570",
-        "label": "FRESNO VA MEDICAL CENTER"
+        "label": "Fresno VA Medical Center"
       }
     ],
     "WY": [
       {
         "value": "568HA",
-        "label": "NEWCASTLE"
+        "label": "Newcastle VA Clinic"
       },
       {
         "value": "666GF",
-        "label": "ROCK SPRINGS CBOC"
+        "label": "Rock Springs VA Clinic"
       },
       {
         "value": "666GB",
-        "label": "CASPER CBOC"
+        "label": "Casper VA Clinic"
       },
       {
         "value": "666GC",
-        "label": "RIVERTON VA CLINIC"
+        "label": "Riverton VA Clinic"
       },
       {
         "value": "666GD",
-        "label": "CODY VA CLINIC"
+        "label": "Cody VA Clinic"
       },
       {
         "value": "666GE",
-        "label": "GILLETTE VA CLINIC"
+        "label": "Gillette VA Clinic"
       },
       {
         "value": "442MB",
         "label": "IDES SHERIDAN VAMC"
       },
       {
         "value": "442HK",
-        "label": "WHEATLAND VA MOBILE CLINIC"
+        "label": "Wheatland VA Mobile Clinic"
       },
       {
         "value": "666QA",
-        "label": "AFTON VA CLINIC"
+        "label": "Afton VA Clinic"
       },
       {
         "value": "666QB",
-        "label": "EVANSTON VA CLINIC"
+        "label": "Evanston VA Clinic"
       },
       {
         "value": "666QC",
-        "label": "WORLAND VA CLINIC"
+        "label": "Worland VA Clinic"
       },
       {
         "value": "442QD",
-        "label": "LARAMIE VA MOBILE CLINIC"
+        "label": "Laramie VA Mobile Clinic"
       },
       {
         "value": "666",
-        "label": "SHERIDAN VA MEDICAL CENTER"
+        "label": "Sheridan VA Medical Center Outbased Benefit Office"
```

Comment:
Let's confirm this name change.

This seems to be referencing another Benefits Office.

Reply:
This should be: **Sheridan VA Medical Center** (https://www.va.gov/find-locations/facility/vha_666).